### PR TITLE
90-upgrade-apollo

### LINKED ISSUE #90: Upgrade apollo, remove redux
## Acceptance Criteria

- [ ] It should use the latest version of apollo
- [ ] It should remove redux

### PR DESCRIPTION
**Pull request by Lepozepo**
Fixes #90 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [ ] Run `test` and make sure all tests pass
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review function and form on Web
- [ ] Review new test logic

### CODE REVIEW

**Comment by mary-pr-poppins** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  No Changelog changes!
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  :exclamation: Big PR
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Lepozepo** [comment]
@conrad-vanl does something like this make sense to you?

File: src/@data/withHomeFeed/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { graphql } from 'react-apollo';
+import { FOLLOWABLE_TOPICS } from '@data/constants';
+import homeFeedQuery from './homeFeedQuery';
+
+export default graphql(homeFeedQuery, {
+  options: () => ({
+    variables: {
+      filters: ['CONTENT'],
+      options: JSON.stringify({ content: { channels: FOLLOWABLE_TOPICS } }),
```
Comment (conrad-vanl) [inline src/@data/withHomeFeed/index.js]: Does this really need to be `stringified`?

**Comment by conrad-vanl** [comment]
Looks good! One question above. Feel free to merge, but would like to know why that needs to be like that.

File: src/@data/withHomeFeed/index.js (hunk 1 above)
Comment (Lepozepo) [reply src/@data/withHomeFeed/index.js]: I'm not sure actually, that is pretty much a copy/paste of what they have in Holtzman. Worth a shot though :D

Comment (Lepozepo) [reply src/@data/withHomeFeed/index.js]: I'll check tomorrow and let you know

**Comment by conrad-vanl** [comment]
FYI, deploys on netlify are failing since adding in the root imports. I fixed it in #87. It's an optional check so you can still merge with it failing.

File: src/@data/withHomeFeed/index.js (hunk 1 above)
Comment (Lepozepo) [reply src/@data/withHomeFeed/index.js]: @conrad-vanl yeah, it does need to be stringified. I'm going to explore `apollo-link-state` more before merging

**Pull request merged**